.github/workflows/ROMROM-CHANGELOG-UPDATE.yaml

```
@@ -359,4 +364,50 @@ jobs:
       - name: 정리
         run: |
-          rm -f pr_content.html summary_section.html raw_summary.txt parse_changelog.py 
\ No newline at end of file
+          rm -f pr_content.html summary_section.html raw_summary.txt parse_changelog.py 
+
+      - name: 자동 PR Merge (CHANGELOG 업데이트 완료 후)
+        if: env.SUMMARY_FOUND == 'true'
+        run: |
+          PR_NUMBER="${{ github.event.pull_request.number }}"
+          echo "🔄 PR #$PR_NUMBER 자동 merge 시작..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)